File: sheet 8 L. Sereja and Dima/main.js
var numberOfCards = +readline()
var cards = readline().split` `.map(x => +x)
var sereja = 0, dima = 0;
var turn = true; // true = sereja, false = dima
while (cards.length) {
    if (turn) {
        if (cards[0] > cards[cards.length - 1]) {
            sereja += cards.shift()
        } else {
            sereja += cards.pop()
        }
        turn = !turn
    } else {
        if (cards[0] > cards[cards.length - 1]) {
            dima += cards.shift()
        } else {
            dima += cards.pop()
        }
        turn = !turn
    }
}
 
print(`${sereja} ${dima}`)

File: sheet 8 L. Sereja and Dima/main.py
_ = input()
nums = [int(i) for i in input().split()]
sereja, dima = 0, 0
turn = True

while nums:
    num = max(nums[0], nums[-1])
    nums.remove(num)
    
    if turn:
        sereja += num
    else:
        dima += num
        
    turn = not turn
    
print(sereja, dima)